link/src/main/java/com/stripe/android/link/serialization/PopupPayload.kt

```diff
@@ -113,7 +122,7 @@ internal data class PopupPayload(
                 ),
                 customerInfo = CustomerInfo(
                     email = customerEmail,
-                    country = customerBillingCountryCode,
+                    country = customerBillingCountryCode ?: merchantCountryCode,
```

Comment:
Is this really what we want to do? This doesn't seem right.

Reply:
The popup needs _something_ here. So we just take our best guess. They use it for defaults on things like the phone number drop down, it's not anything that's "saved", just to help get the user started filling out their info.

Reply:
If this is what iOS does, then let's go with this.